src/content/docs/browser/new-relic-browser/configuration/group-browser-metrics-urls.mdx

```diff
@@ -39,11 +39,10 @@ You can modify how browser groups your data by adding to your application's URL
 
 Order is not important when listing multiple segments. However, make sure the segments you add **exactly match** the way the segments appear in your URLs. If a URL path segment ends in `.html` or `.jsp`, the extension must be included in the segment. Segments are case-sensitive.
 
-To add new URL segments to your application's allow list:
+To add new URL segments to your application's allow list, you can add them programmatically using [GraphQL](/docs/apis/nerdgraph/examples/browser-monitoring-config-nerdgraph#browser-segments), or you can use segment allow lists in the New Relic browser monitoring UI like this:
```

Comment:
I'm not sure the "like this" is completely necessary here, but it's just a small and probably inconsequential change to make

Reply:
@jeff-colucci good catch! I dropped that!